apisix/core/log.lua

```diff
@@ -42,17 +43,27 @@ local log_levels = {
 }
 
 
-local cur_level = ngx.config.subsystem == "http" and
-                  require "ngx.errlog" .get_sys_filter_level()
+local cur_level
+
 local do_nothing = function() end
 
 
+local function update_log_level()
+    -- Nginx use `notice` level in init phase instead of error_log directive config
+    -- Ref to src/core/ngx_log.c's ngx_log_init
+    if ngx.get_phase() ~= "init" then
+        cur_level = ngx.config.subsystem == "http" and ngx_errlog.get_sys_filter_level()
```

Comment:
Shall we remove `ngx.config.subsystem == "http"` ?

Reply:
Yes, we can do it in the next PR.